### DISCUSSION TITLE
How to create Custom ServiceHostFactory in CoreWCF

### DISCUSSION THREAD

**Discussion by pascalrebello** [CoreWCF/CoreWCF#1159] [Q&A]
How to create Custom ServiceHostFactory in CoreWCF

In the case of WCF we can extend it by using ServiceHostFactory and SeviceHost.

And finally mentioning the Factory in the .SVC file like this (<% @ ServiceHost Language="C#" Service="WcfService1.Service1"
Factory="WcfService1.MyWebServiceHostFactory" %>

So similar way how we can implement this in CoreWCF? Kindly share your valuable suggestions.

**Comment by g7ed6e** [comment]
> How to create Custom ServiceHostFactory in CoreWCF
> 
> In the case of WCF we can extend it by using ServiceHostFactory and SeviceHost.
> 
> And finally mentioning the Factory in the .SVC file like this (<% @ ServiceHost Language="C#" Service="WcfService1.Service1" Factory="WcfService1.MyWebServiceHostFactory" %>
> 
> So similar way how we can implement this in CoreWCF? Kindly share your valuable suggestions.

@pascalrebello There is no such SVC file with CoreWCF. Could you clarify the goal you are trying to reach using a custom ServiceHostFactory ?
